From 370a3695aa3e7e99e0b69489c8ac114025130e1d Mon Sep 17 00:00:00 2001
From: Subramaniam Chanderashekarapuram <c.a.subramaniam@gmail.com>
Date: Wed, 24 Jun 2020 17:14:22 -0400
Subject: [PATCH] support for buildroot builds

Signed-off-by: Subramaniam Chanderashekarapuram <c.a.subramaniam@gmail.com>
---
 CMakeLists.txt | 13 +++----------
 1 file changed, 3 insertions(+), 10 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 3aedb246f..423a05123 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,5 +1,5 @@
-#set to minimum version that supports clean build
-cmake_minimum_required(VERSION 3.16.0)
+#set to minimum version that supports clean build on cygwin
+cmake_minimum_required(VERSION 2.8.4)
 
 project(domoticz)
 
@@ -609,14 +609,7 @@ ELSE(USE_STATIC_BOOST)
    message(STATUS "Linking against boost dynamic libraries")
 ENDIF(USE_STATIC_BOOST)
 
-find_package(Boost REQUIRED COMPONENTS thread system)
-IF(Boost_FOUND)
-    MESSAGE(STATUS "BOOST includes found at: ${Boost_INCLUDE_DIR}")
-ELSE(Boost_FOUND)
-    MESSAGE(FATAL_ERROR "Boost thread/system library not found on your system, try to get this installed.")
-ENDIF(Boost_FOUND)
-
-target_link_libraries(domoticz Boost::thread Boost::system)
+find_package(Boost REQUIRED COMPONENTS thread system atomic)
 
 # compare found vs required libBoost version
 IF(Boost_VERSION VERSION_LESS DOMO_MIN_LIBBOOST_VERSION)
-- 
2.17.1

